test/utils.test.ts

```diff
@@ -23,3 +23,69 @@ describe("Regex creator", () => {
     expect("question").not.toMatch(re1);
   });
 });
+
+describe("Get files changed", () => {
+  it("Should return array of files changed", async () => {
+    const mockOctokit = {
+      rest: {
+        pulls: {
+          listFiles: jest.fn().mockResolvedValue({
+            data: [
+              { filename: "file1.ts" },
+              { filename: "file2.ts" },
+              { filename: "file3.ts" }
+            ]
+          })
+        }
+      }
+    };
+    
+    const result = await getFilesChanged(mockOctokit, "owner", "repo", 123);
+    expect(result).toHaveLength(3);
+    expect(result[0].filename).toBe("file1.ts");
+    expect(mockOctokit.rest.pulls.listFiles).toHaveBeenCalledWith({
+      owner: "owner",
+      repo: "repo",
+      pull_number: 123
+    });
+  });
+
+  it("Should return different file arrays", async () => {
+    const mockOctokit = {
+      rest: {
+        pulls: {
+          listFiles: jest.fn().mockResolvedValue({
+            data: [
+              { filename: "file1.ts" },
+              { filename: "file2.ts" },
+              { filename: "file3.ts" },
+              { filename: "file4.ts" },
+              { filename: "file5.ts" }
+            ]
+          })
+        }
+      }
+    };
+    
+    const result = await getFilesChanged(mockOctokit, "owner", "repo", 456);
+    expect(result).toHaveLength(5);
+    expect(result[4].filename).toBe("file5.ts");
+  });
+
+  it("Should return large file arrays", async () => {
```

Comment:
This is not testing any specific functionality; if it fails, it fails in the physical infrastructure, so it's not really needed.